src/kvp/annotation/mod.rs

```diff
@@ -45,13 +44,14 @@ pub type AnnotationError = KeyValuePairError<Infallible>;
 #[derive(Debug)]
 pub struct Annotation(KeyValuePair<AnnotationValue>);
 
-impl<T> TryFrom<(T, T)> for Annotation
+impl<T, K> TryFrom<(T, K)> for Annotation
 where
     T: AsRef<str>,
+    K: AsRef<str>,
```

Comment:
nit: maybe we want to use `K` and `V` generics?

Reply:
_same applies for labels_

Reply:
Sure, I will change that.

Reply:
This might be a little tricky, for example:

https://github.com/stackabletech/operator-rs/blob/008056d743afc0d9acdb04b721a84fe495f6b7e2/src/kvp/mod.rs#L167-L184

`V` is already used as the final value type.

Reply:
Ah ok, personally I would do:

```rust
impl<const N: usize, T, K, V> TryFrom<[(K, V); N]> for KeyValuePairs<T>
 where 
     K: AsRef<str>, 
     V: AsRef<str>, 
     T: Value + std::default::Default, 
```

But will leave that up to you. It's not a blocker in my eyes.

Reply:
Oh that's a nice solution. I will update this.

Reply:
Changed in 0e5ca4d